src/components/views/settings/tabs/user/PreferencesUserSettingsTab.js

```diff
@@ -161,6 +175,14 @@ export default class PreferencesUserSettingsTab extends React.Component {
                 label={_t('Start automatically after system login')} />;
         }
 
+        let warnBeforeExitOption = null;
+        if (this.state.warnBeforeExitSupported) {
+            warnBeforeExitOption = <LabelledToggleSwitch
+                value={this.state.warnBeforeExit}
+                onChange={this._onWarnBeforeExitChange}
+                label={_t('Warn before quitting')} />;
+        }
+
```

Comment:
I think these sorts of settings are what Design would typically want to take a look at, given we try to not have too many options and instead try to design things that make sense instead.

I can see this being potentially controversial to turn on, so a setting would be appreciated by me at least.

Reply:
I have run that through @nadonomy who suggested to add that option in "Settings > Preferences > General"

I believe the copy is very much up for debate at this point